change/@ni-nimble-components-51de64d0-1537-4dce-89b5-e85857542ebb.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I tested on macOS Firefox and Safari and everything looks great. The scrolling animation is a nice improvement. Trackpad scrolling works well too.

On iOS Safari and Firefox things mostly worked well too, including pan scrolling and momentum scrolling. I saw a minor bug where tapping the scroll buttons while a scroll animation was still in progress would allow you go past the end of the last tab, showing blank space while the tabs scrolled out of the viewport and are no longer visible. I don't think that's worth investigating: you can restore your state easily by scrolling back.